ingesters/HttpIngester/hec_config.go

```diff
@@ -39,6 +40,7 @@ type hecCompatible struct {
 	Tag_Name                  string   //the tag to assign to the request
 	Ignore_Timestamps         bool
 	Timestamp_Format_Override string //override the timestamp format (only used for raw)
+	Timestamp_Cutoff          string
```

Comment:
same, i say durations not timestamps.  and maybe it should be global, not per listener